src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1589,6 +1597,50 @@ def use_radamsa_mutator_plugin(extra_env):
   return True
 
 
+def use_peach_mutator(extra_env, grammar):
+  """Decide whether or not to use peach mutator, and set up all of the
+  environment variables necessary to do so."""
+  # TODO(mpherman): Include architecture info in job definition and exclude
+  # i386.
+  if not is_linux_asan():
+    return False
+
+  if not grammar:
+    return False
+
+  pit_path = pits.get_path(grammar)
+
+  if not pit_path:
+    return False
+
+  # Set title and pit environment variables
+  extra_env['PIT_PATH'] = pit_path
+  extra_env['PIT_TITLE'] = grammar
+
+  # Extract zip of peach mutator code.
+  peach_dir = os.path.join(environment.get_platform_resources_directory(),
+                           'peach')
+  unzipped = os.path.join(peach_dir, 'mutator')
+  source = os.path.join(peach_dir, 'peach_mutator.zip')
```

Comment:
why do we keep it in zip and extract every time?

Reply:
Because the peach code has to be python 2, and we don't want python 2 code in the repo. @mbarbella-chromium and I figure this would be the best way around that.

Reply:
ack